pyfr/solvers/base/elements.py

```diff
@@ -1,11 +1,26 @@
-from functools import cached_property
+from functools import cached_property, wraps
 
 import numpy as np
 
 from pyfr.nputil import npeval, fuzzysort
 from pyfr.util import memoize
 
 
+def inters_map(meth):
+    @wraps(meth)
+    def newmeth(self, eidx, fidx):
+        nfp = self.nfacefpts[fidx]
+        cmap = (eidx,)*nfp
+
+        mid, rmap, *lda = meth(self, eidx, fidx)
+
+        if lda:
+            return (mid,)*nfp, rmap, cmap, (lda)*nfp
```

Comment:
What is the extra argument?

Reply:
It's the stride for the vector interface terms. Before, I was doing `(LDA,)`, which returned `[]` when it was empty, but when it's not empty the concatenated array becomes 2D. Which in some case leads to huge amounts of memory needing to be allocated

Reply:
But this code now returns four things?  It does not make sense.